compiler/qsc_codegen/src/qir.rs

```diff
@@ -60,6 +61,24 @@ pub fn fir_to_qir(
     Ok(ToQir::<String>::to_qir(&program, &program))
 }
 
```

Comment:
this is a documentation-light file, but perhaps it'd be good to document at least newly-added `pub` fns